### ISSUE TITLE
[BUG] `fill-current` not recognized

### ISSUE THREAD

**Issue by thebuilder** [francoismassart/eslint-plugin-tailwindcss#95]
```
warning  Classname 'fill-current' is not a Tailwind CSS class!  tailwindcss/no-custom-classname
```

The `fill-current` class should be allowed:)
https://tailwindcss.com/docs/fill

**Comment by francoismassart** [comment]
Which version of the plugin are you using ?

I added a test on my local copy of v2 & v3 and they both pass.

Maybe it is due to your tailwind.config.js file's content

**Comment by thebuilder** [comment]
Hey, that's weird - Was trying to upgrade upgrade project to the latest version of both Tailwind and the ESLint plugin, and this one popped up as a warning.

There's nothing related to `fill` or `current` in the tailwind config file. I contains a `theme` and some official plugins.

**Comment by thebuilder** [comment]
I tried deleting the `theme` from my `tailwind.config.js`, and that did indeed clear that warning. Interesting.

Debugging a bit, seems to be this specfic value in my config:

```js
{
    extend: {
      lineHeight: {
        title: 0.9,
      },
    },
}
```

**Comment by francoismassart** [comment]
If you are using the ESLint extension, make sure you restart VSCode after the update...

**Comment by thebuilder** [comment]
This is running `eslint` in the terminal.

**Comment by thebuilder** [comment]
I can get around it with `whitelist`, but still interesting what could cause this? A sideeffect of extending theme?

**Comment by francoismassart** [comment]
I added these tests but I can't reproduce the issue:
![image](https://user-images.githubusercontent.com/704026/149333379-62312156-fdcc-4e34-b448-bef5e28f80d8.png)

Can you try this:

1. `rm -rf node_modules/`
2. `npm i`
3. Restart VSCode
4. Retry linting

**Comment by francoismassart** [comment]
using `3.2.0`

**Comment by francoismassart** [comment]
@thebuilder As I cannot reproduce this issue, please provide a small demo repo on which I could reproduce the bug.
If you can, please re-open this issue with the link to the repo
